assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
+
+According to the New York Times [4, page 7], documents released by House investigators reveal that the ‘testing bulletin’ showed that the website as of September 30 could handle only about 1100 users at a time, even though officials had said it should have been abble to accomodate perhaps as many as 60,000 users.
+
+######After the launch
+During its launch (1 October 2013) the website did not perform well. Only 271,000 of the 9.47 million users that try to registrate where succesfull. ?There was already $700 million dollar spend? In 2014 the total number of costs of the project is 1.7 billion dollar (reference). Intuitively I ask myself what would have happened if they had established a schedule of access to the site?. There were official warnings telling that the site was going to be unable to accommodate so many users at once.
+
+An extra team is recluted (22th October 2013) It was recognized that product testing was insufficient. Serious underestimation of time.
+
+QSSI is selected as the contractor responsible to now oversee federal website fixes (October 27, 2013). A single data center brings down the mission-critical website and should be presumed and avoided. Such requirements should be identiﬁed and addressed before project completion.
+
+(October 31, 201) Again an unrealistic schedule, site crash for second time due small amount of servers, an explicit gap at project definition and planning.
+
+(November 14, 2013) President Obama declares to press that he wasn’t good inform of the situation. The project team was not prepared for the failure.
+
+(November 30, 2013)  Enrollment statistics show that 137,000 individuals had signed up for plans using the site compared to the 227,000 that had enrolled through the 14 state run exchanges.
+
+Obama Administration set deadline of a working site for the “vast majority” of users. (December 1, 2013)
+
+
+### Failure causes & possible solutions.
+The failure was caused by a complex interplay of problems:
+
+##### Cause 1: Contractor Management.
+ 6. Contractor delays and performance issues were not always identified.
+ 7. A contractor incurred unauthorized costs that increased the cost of the contract.
+ 8. Contracting officers in all Government agencies did not have access to contractor past-performance evaluations when making contract awards.
+ 9. Critical deliverables and management decisions were not properly documented.
+ 
+> At this excerpt extracted from [source] we add the question: **where could it change?**. A stricter monitoring of the deliverables by the contractors (with small and testable deliverables), could have helped this project.
+
+
+##### Cause 2: Architecture, non functional requirements.
+ 1. The system was not designed to handle the massive influx of initial users. 
+ 2. Insufficient effort made to build a system that would meet the performance and availability needs of it's stakeholders. 
```

Comment:
"Security solutions were seemingly slapped together in a shoddy manner instead of through the kind of systematic approach that is expected in a highquality software solution."
This sentence is a bit hard to read in the end. I think you can replace "instead of through the kind of systematic approach" with "instead of through a systematic approach" to make it a bit better.